packages/fileeditor-extension/src/commands.ts

```diff
@@ -904,11 +906,28 @@ export namespace Commands {
       }
     });
 
+    commands.addCommand(CommandIDs.dismissCompleter, {
+      label: trans.__('Dismisses the completion wideget.'),
+      execute: () => {
+        const id =
+          editorTracker.currentWidget && editorTracker.currentWidget.id;
+        if (id) {
+          return manager.dismiss(id);
+        }
+      }
+    });
+
     commands.addKeyBinding({
       command: CommandIDs.selectCompleter,
       keys: ['Enter'],
       selector: '.jp-FileEditor .jp-mod-completer-active'
     });
+
+    commands.addKeyBinding({
+      command: CommandIDs.dismissCompleter,
+      keys: ['Escape'],
+      selector: '.jp-FileEditor .jp-mod-completer-active'
+    });
```

Comment:
We will need to repeat this for the two other cases: console and notebook, see https://github.com/search?q=repo%3Ajupyterlab%2Fjupyterlab%20CommandIDs.selectCompleter&type=code

I think that we do not need to call `commands.addKeyBinding` if we are adding shortcuts using JSON schema (`jupyter.lab.shortcuts`).